backend/src/data/migrations/20220906143050_add_course_categories_prices_table.ts

```diff
@@ -0,0 +1,71 @@
+import { Knex } from 'knex';
+
+import { IdContainer } from '~/common/types/types';
```

Comment:
I think a migration file shouldn't import other things.

Reply:
I needed type for the id containers but did not want to duplicate types. Chance for this type to change is very low, so I decided to use that. Let's see, what coaches are going to advice

Reply:
Even if type is common and doesn't look unstable or modifiable - it's better to have it duplicated in migration, then in one day face an error in migrations

Reply:
Ok, got it